mypyc/test-data/irbuild-isinstance.test

```diff
@@ -0,0 +1,97 @@
+[case testIsinstanceInt]
+def is_int(value):
+    return isinstance(value, int)
+
+[out]
+def is_int(value):
+    value, r0 :: object
+    r1 :: int32
+    r2 :: bit
+    r3 :: bool
+    r4 :: object
+L0:
+    r0 = load_address PyLong_Type
+    r1 = PyObject_IsInstance(value, r0)
+    r2 = r1 >= 0 :: signed
+    r3 = truncate r1: int32 to builtins.bool
+    r4 = box(int32, r1)
+    return r4
+
+[case testIsinstanceNotBool1]
+def is_not_bool(value):
+    return not isinstance(value, bool)
```

Comment:
Here adding a type annotation to the function could be more useful, since the IR is pretty verbose right now.